chibuezeayogu/Events-Manager
# #159770703 Implement user signup

**What does the PR do?**
- This PR implements user `sign up`

**Description of Task to be completed?**
- Ensure the users can sign up
- Ensure that token is generated when users sign's up 
- Ensure test is passing 

**How should this be manually tested?**
- Clone the repository [Events-Manager](https://github.com/chibuezeayogu/Events-Manager.git) and
- Checkout to  this branch `ft-user-signup-#159770703`
- run `bundle install` to install new `rails gems`
- run `rails server` and test this route ` Post /api/vi/users/signup` using `Postman`
- run `rspec` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#159770703](https://www.pivotaltracker.com/story/show/159770703)

**Screenshots**
<img width="1354" alt="screen shot 2018-08-23 at 3 33 12 pm" src="https://user-images.githubusercontent.com/26305570/44531979-f4d52400-a6e9-11e8-83dc-e0ef7199f20b.png">

<img width="1352" alt="screen shot 2018-08-23 at 3 32 44 pm" src="https://user-images.githubusercontent.com/26305570/44532011-04ed0380-a6ea-11e8-8980-d13af7cbb216.png">
